hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java

```diff
@@ -1245,8 +1245,49 @@ public void testBalancerCliParseWithWrongParams() {
       Balancer.Cli.parse(parameters);
       fail(reason + " for -source parameter");
     } catch (IllegalArgumentException ignored) {
-      // expected
+
+    }
+
+    parameters = new String[] {"-excludeSource"};
```

Comment:
Thank you for adding a unit test around CLI argument parsing. Please add additional unit test coverage for the functionality you are adding. I would expect tests to cover all three flags, happy cases, as well as corner cases.

Reply:
Hey @ctrezzo I have added unit tests, please take a look.